docs/project/2024/ioana_miruna.badica/index.md

```diff
@@ -0,0 +1,85 @@
+# Smart Lamp
+The prototype of a lamp with multiple functionalities.
+
+:::info 
+
+**Author**: Bădică Ioana-Miruna <br>
+**GitHub Project Link**: [Smart Lamp](https://github.com/jojomojo22/upb-fils-ma.github.io/blob/project_documentation/docs/project/2024/ioana_miruna.badica/index.md)
+
+:::
+
+## Description
+
+The project consists of a LED that turns on the light automatically when it detects movement and only when it is dark. Besides, it should still have its regular "lamp" function, so it will have a button to turn it on "smart" mode. 
+
+## Motivation
+
+I have chosen this project because I wanted to implement something that could turn into a bigger project later such as using a real lamp instead of a LED, or implementing new functions like (if I were to use a RGB LED) to add a remote controller to switch the colors of the light from far away.
+
+## Architecture 
+
+Components <br>
```

Comment:
Please use the correct markdown syntax for lists.

Reply:
I solved the <br> problem in all places, I hope it is okay now. Also filled the pull request template. Please let me know if I should make any other changes. Thank you!